content/testsuite/testsuite.go

```diff
@@ -50,6 +50,10 @@ func makeTest(t *testing.T, name string, storeFn func(ctx context.Context, root
 	}
 }
 
+var labels = map[string]string{
+	"containerd.io/gc.root": time.Now().UTC().Format(time.RFC3339),
```

Comment:
Do we need nanos here?

Reply:
No, nanos are not needed. This value is not checked and for consistency with what clients set